cmd/microindex/convert/command.go

```diff
@@ -49,6 +50,7 @@ func newCommand(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
 	f.IntVar(&c.frameThresh, "f", 32*1024, "minimum frame size used in microindex file")
 	f.StringVar(&c.outputFile, "o", "index.zng", "name of microindex output file")
 	f.StringVar(&c.keys, "k", "", "comma-separated list of field names for keys")
+	f.BoolVar(&c.desc, "desc", false, "specify data is in descending order")
```

Comment:
Nit: Put this between frameThresh and outputFile so these are in the same order as the fields of Command.

Reply:
yup